src/accountant.ts

```diff
@@ -37,38 +37,120 @@ export class Accountant {
     }
 
     async run(): Promise<void> {
+        await this.initClient()
+
         if(this.claimsCheckOnly.length > 0) {
-          for (let i = 0; i < this.claimsCheckOnly.length; i++) {
-            this.logger.info(`Processing checkOnlyClaim ${i} for ${this.claimsCheckOnly[i].alias}`);
-            await this.processClaimCheckOnly(this.claimsCheckOnly[i]);
-          }
-          this.client.disconnect();
-          return
+          this.logger.info(`Processing claims checking them only...`)
+          await this.processClaimsCheckOnly()
         }
         if (this.claims.length > 0) {
-            for (let i = 0; i < this.claims.length; i++) {
-                this.logger.info(`Processing claim ${i} for ${this.claims[i].alias}`);
-                await this.processClaim(this.claims[i]);
-            }
+          this.logger.info(`Processing claims...`)
+          await this.processClaims()
         }
         if (this.claimThirdParty?.targets.length > 0) {
-          const promiseArray = []
-          for (let i = 0; i < this.claimThirdParty.targets.length; i++) {
-              this.logger.info(`Processing third party claim ${i} for ${this.claimThirdParty.targets[i].alias}`);
-              promiseArray.push(this.processClaimThirdParty(this.claimThirdParty.claimerKeystore,this.claimThirdParty.targets[i]));
-          }
-          await Promise.all(promiseArray)
+          this.logger.info(`Processing third party claims...`)
+          await this.processClaimsThirdParty()
         } 
         if (this.transactions.length > 0) {
-            for (let i = 0; i < this.transactions.length; i++) {
-                this.logger.info(`Processing tx ${i} from ${this.transactions[i].sender.alias} to ${this.transactions[i].receiver.alias}`);
-                await this.processTx(this.transactions[i]);
-            }
+          this.logger.info(`Processing transfers...`)
+          await this.processTransfers()
         }
+        
         this.client.disconnect();
     }
 
-    private async processTx(tx: Transaction): Promise<void> {
+    private async initClient(): Promise<void> {
+      await this.client.api()
```

Comment:
There's no point in having a single line function, save those 3 extra lines.

Reply:
it is no more a single line function